storybook/src/components/Card/Card.docs.mdx

```diff
@@ -14,16 +14,17 @@ import README from "../../../../packages/css/src/components/card/README.md?raw";
 
 ### With tagline
 
-A card can display a tagline above the heading, a short term like a category or type of information.
-Wrap the tagline and the heading in a `Card.HeadingGroup`.
-This ensures screen readers first read the heading and then the tagline.
+A Card can display metadata in a tagline above the heading, like a category or content type.
+Wrap the Heading in a Heading Group and set its `tagline` property.
+This ensures screen readers read out the heading before the metadata.
 
 <Canvas of={CardStories.WithTagline} />
 
 ### With image
 
-A card often displays the [image](/docs/components-media-image--docs) of the detail page.
-It must use one of the available [aspect ratios](/docs/brand-design-tokens-aspect-ratio--docs).
+A Card often displays the image of the article it links to.
+The Card Image subcomponent renders an [Image](/docs/components-media-image--docs) component and accepts its properties, e.g. to set an aspect ratio or provide a set of image sources.
+An `alt` text is almost never necessary for the image of a card.
```

Comment:
Maybe we should be a bit more strict here. If the image does have an `alt` text, it will not be read by screen readers when you navigate to the link or heading. Maybe we should even hardcode `alt=""` in CardImage and omit `alt` from the type? I'd keep this text though, you can also use the Next image component here.

```suggestion
Only use decorative images with `alt=""` here. Screen readers will not read the `alt` text of non-decorative images when navigating to a Card. 
```